Framework/PythonInterface/mantid/geometry/src/Exports/PointGroup.cpp

```diff
@@ -40,6 +42,14 @@ boost::python::list getEquivalents(PointGroup &self, const object &hkl) {
 Mantid::Kernel::V3D getReflectionFamily(PointGroup &self, const object &hkl) {
   return self.getReflectionFamily(Converters::PyObjectToV3D(hkl)());
 }
+
+std::string __repr__implementation(const PointGroup &self) {
+  std::stringstream ss;
+  ss << "PointGroupFactory.createPointGroup(\"";
```

Comment:
I agree. I faced this dilemma when writing this. I as `__repr__` should ideally be able to reproduce the object I went this as that is how `PointGroup`s are currently created via the python api. In future perhaps we could add constructors that call the factory cpp side.